pkg/rulesConfig/defaultRules/10-ensure-sca-scanner.go

```diff
@@ -0,0 +1,83 @@
+package defaultRules
```

Comment:
almost all code here except for the regexes is same the same in EnsureIacScanner.

please add TechDebt to combine the two.